specification/communication/data-plane/PhoneNumbers/preview/2021-10-25-preview/phonenumbers.json

```diff
@@ -0,0 +1,1639 @@
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "in": "body",
+            "name": "body",
+            "description": "The phone number search request.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/PhoneNumberSearchRequest"
+            }
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "The service has accepted the request and will start processing later. It will return 'Accepted' immediately and include an 'Operation-Location' and 'Location' headers. Client side should further query the operation status using the URL specified in 'Operation-Location' header. When operation completes successfully query final result using URL specified in 'Location' header.",
+            "schema": {
+              "$ref": "#/definitions/PhoneNumberSearchResult"
+            },
+            "headers": {
+              "Location": {
+                "description": "URL to retrieve the final result after operation completes.",
+                "type": "string"
+              },
+              "Operation-Location": {
+                "description": "URL to query for status of the operation.",
+                "type": "string"
+              },
+              "operation-id": {
+                "description": "The operation id.",
+                "type": "string"
+              },
+              "search-id": {
+                "description": "The search operation id.",
+                "type": "string"
+              }
+            }
+          },
+          "default": {
+            "description": "Failure",
+            "schema": {
+              "$ref": "../../../Common/stable/2021-03-07/common.json#/definitions/CommunicationErrorResponse"
+            }
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-long-running-operation-options": {
+          "final-state-via": "location"
+        },
+        "x-ms-examples": {
+          "Search for phone numbers": {
+            "$ref": "./examples/SearchPhoneNumbers.json"
+          }
+        }
+      }
+    },
+    "/availablePhoneNumbers/searchResults/{searchId}": {
+      "get": {
+        "tags": [
+          "Find and purchase available phone numbers"
+        ],
+        "summary": "Gets a phone number search result by search id.",
+        "operationId": "PhoneNumbers_GetSearchResult",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "path",
+            "name": "searchId",
+            "description": "The search Id.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/PhoneNumberSearchResult"
```

Comment:
Is this pageable?  I'd expect it to page because you allow up to int.MaxValue numbers to be requested, but I don't see a nextLink anywhere in the result?